projects/cdk/types/date-mode.ts

```diff
@@ -0,0 +1 @@
+export type TuiDateMode = 'dd.mm.yyyy';
```

Comment:
Could you clarify the purpose of this? Do you want to make multiple modes in the future like `dd.mm.yyyy | mm.dd.yyyy`?

Reply:
i've found this [TODO in the code](https://github.com/TinkoffCreditSystems/taiga-ui/blob/main/projects/cdk/date-time/date-fillers.ts#L4):

> TODO: think about yyyy.mm.dd format

So, multiple modes will be in the future